docs/style-guide.adoc

```diff
@@ -237,73 +287,81 @@ WITH null AS n1, null as n2
 RETURN n1 IS NULL && n2 IS NOT NULL
 ----
 
-. When patterns wrap lines, break after arrows, not before.
+. Write boolean literals in lower case.
 +
 .Bad
 [source, cypher]
 ----
-MATCH (:Person)-->(vehicle:Car)-->(:Company)
-      <--(:Country)
-RETURN count(vehicle)
+WITH TRUE AS b1, False AS b2
+RETURN b1 && b2
 ----
 +
 .Good
 [source, cypher]
 ----
-MATCH (:Person)-->(vehicle:Car)-->(:Company)<--
-      (:Country)
-RETURN count(vehicle)
+WITH true AS b1, false AS b2
+RETURN b1 && b2
 ----
 
-. Surround node patterns with parentheses.
-.. This only concerns old Cypher code, as omitting parentheses is no longer legal in Cypher.
+. Use camel case, starting with a lower case character, for:
+.. functions
+.. properties
+.. variables
+.. parameters
 +
 .Bad
 [source, cypher]
 ----
-MATCH person-->vehicle
-RETURN person, vehicle
+CREATE (N {Prop: 0})
+WITH RAND() AS Rand, $pArAm AS MAP
+RETURN Rand, MAP.property_key, Count(N)
 ----
 +
 .Good
 [source, cypher]
 ----
-MATCH (person)-->(vehicle)
-RETURN person, vehicle
+CREATE (n {prop: 0})
+WITH rand() AS rand, $param AS map
+RETURN rand, map.propertyKey, count(n)
 ----
 
-. Avoid having to use back-ticks to escape characters and keywords.
+=== Patterns
+
+. When patterns wrap lines, break after arrows, not before.
 +
 .Bad
 [source, cypher]
 ----
-MATCH (`odd-ch@racter$`:`Spaced Label` {`&property`: 42})
-RETURN labels(`odd-ch@racter$`)
+MATCH (:Person)-->(vehicle:Car)-->(:Company)
+      <--(:Country)
+RETURN count(vehicle)
 ----
 +
 .Good
 [source, cypher]
 ----
-MATCH (node:NonSpacedLabel {property: 42})
-RETURN labels(node)
+MATCH (:Person)-->(vehicle:Car)-->(:Company)<--
+      (:Country)
+RETURN count(vehicle)
 ----
 
-. Do not use a semicolon at the end of the statement.
+. Surround node patterns with parentheses.
```

Comment:
I wonder whether we should zap this section (surrounding node patterns with parentheses) entirely? I don't think there is much value about talking about old Cypher code here...